### PR TITLE
Rocky9

### PR DESCRIPTION
**Pull request by pwalczysko**
Make tests work on Rocky9 and Ubuntu22.04. 

Also, this PR bumps to python 3.9

cc @khaledk2 @jburel

### CODE REVIEW

**Comment by pwalczysko** [comment]
Tests pass

File: meta/main.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: Manage SSL certificates for web-servers
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.3
+  min_ansible_version: 2.10
```
Comment (jburel) [inline meta/main.yml:7]: are we bumping the version in all the roles even if this role does not require it?
We have not done that for the other roles. Just need to make sure we are all on the same page when updating the roles

Comment (pwalczysko) [reply meta/main.yml:7]: > are we bumping the version in all the roles even if this role does not require it?

Yes, this is what I was doing. Probably I misunderstood. I will revert it to 2.3.

Comment (pwalczysko) [reply meta/main.yml:7]: fixed in https://github.com/ome/ansible-role-ssl-certificate/pull/19/commits/a6adfde7b0f80dd71b0a7dedeb9eb9081f54f6e8

**Comment by jburel** [comment]
Did you test it against our server?

**Comment by pwalczysko** [comment]
> Did you test it against our server?

Yes, I ran the role on the RHEL 9, the testing 134.36.4.3 previously and it worked fine.
I did not change the role itself in this PR, just the testing framework.

I have rerun the playbook now to make sure, I have in the output:

```
...
TASK [ome.ssl_certificate : check parameters] **********************************
skipping: [134.36.4.3]

TASK [ome.ssl_certificate : install openssl] ***********************************
ok: [134.36.4.3]

TASK [ome.ssl_certificate : create certificates directory] *********************
ok: [134.36.4.3] => (item=/etc/ssl/localcerts/server.crt)
ok: [134.36.4.3] => (item=/etc/ssl/localcerts/intermediate.crt)
ok: [134.36.4.3] => (item=/etc/ssl/localcerts/bundled.crt)
ok: [134.36.4.3] => (item=/etc/ssl/localcerts/server.key)

TASK [ome.ssl_certificate : write SSL public certificate] **********************
ok: [134.36.4.3]

TASK [ome.ssl_certificate : write SSL intermediate certificate] ****************
ok: [134.36.4.3]

TASK [ome.ssl_certificate : write SSL certificate key] *************************
ok: [134.36.4.3]

TASK [ome.ssl_certificate : generate self-signed SSL certificate] **************
skipping: [134.36.4.3]

TASK [ome.ssl_certificate : read public certificate] ***************************
ok: [134.36.4.3]

TASK [ome.ssl_certificate : read certificate key] ******************************
ok: [134.36.4.3]

TASK [ome.ssl_certificate : write bundled certificate] *************************
ok: [134.36.4.3]

TASK [ome.ssl_certificate : write SSL combined certificate key] ****************
skipping: [134.36.4.3]
...
```

**Pull request merged**